nss-day-cohort-47/Class-Website
# Mc modal

## Description of Proposed Changes
modals are basically finished
- 

## Steps to Test
take a look at the modal and make sure it looks like the wireframe
```sh
git fetch --all
git checkout mc-modal

```


## Mentions @username

Tag users that need to review this code
@LaurelMorrison , @prestonshotts , @coltmay
